### PR TITLE
[IAM-5434] extend section for sso cert rotation

### PR DESCRIPTION
**Pull request by przemek-sl**
### Description
1. Extend section about Sauce Labs SSO cert rotation. Provide detailed instructions. 
2. Add section about expiration and rotation of identity provider certificate. This was misleading for some customers and they tried to use instructions for the rotation of Sauce Labs certificate. 

### Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation fix (typos, incorrect content, missing content, etc.)

### CODE REVIEW

File: docs/basics/sso/setting-up-sso.md [hunk 1]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
 
 Sauce Labs Service Provider does not require any extra attributes in the SAML Response.
 
-### Certificate rotation
+### Identity Provider Certificate Rotation
+
+SAML certificates have a predetermined expiration date. It is possible that your identity provider will notify you in advance or provide an alert regarding the certificate's expiry.
+
+If the certificate for the Sauce Labs app from your identity provider is nearing expiration or has already expired, please follow these steps:
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/setting-up-sso.md:115]: fixed

File: docs/basics/sso/setting-up-sso.md [hunk 2]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
...
+The certificate is valid for one year. **120 days** before the expiration date the new certificate will be available in Sauce Labs [metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).
+
+During the rotation period the Sauce Labs metadata contain two valid certificates. If you are unsure which certificate to upload, copy the content of the `<ds:X509Certificate>` tag and paste it into [one of the oneline x.509 certificate decoders](https://www.sslshopper.com/certificate-decoder.html). Then, choose the certificate that has a longer validity period.
+
+To convert the certificate from the metadata into the PEM format required by most identity providers, you can utilize the [online tool provided by OneLogin](https://www.samltool.com/format_x509cert.php). Follow these steps:
+
+1. Copy the content of the `<ds:X509Certificate>` tag and paste it into the **X.509 cert** text box. Click **FORMAT X.509 CERTIFICATE**.
+2. Copy the content of the **X.509 cert with header** textbox and save it in a text file, e.g. `saucelabs-sso.pem`.
```
[the remark this answers is not in the document]
Comment (przemek-sl) [reply docs/basics/sso/setting-up-sso.md:135]: fixed

**Review by marcinsauce — APPROVED** [review]

File: docs/basics/sso/setting-up-sso.md [hunk 3]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
...
 
-### Certificate rotation
+### Identity Provider Certificate Rotation
+
+SAML certificates have a predetermined expiration date. It is possible that your identity provider will notify you in advance or provide an alert regarding the certificate's expiry.
+
+If the certificate for the Sauce Labs app from your identity provider is nearing expiration or has already expired, follow these steps:
+
+1. Generate a new certificate for the Sauce Labs app within your identity provider. This new certificate will replace the expired one and will be reflected in the identity provider metadata.
+2. Export or copy the updated metadata file from your identity provider. Then, proceed to upload it in Sauce Labs by following [these steps](/basics/sso/setting-up-sso/#integrating-with-sauce-labs-service-provider).
```
Comment (lysannep) [inline docs/basics/sso/setting-up-sso.md:118]:
```suggestion
2. Export or copy the updated metadata file from your identity provider, then upload it to Sauce Labs. For information on how to upload the metadata file, see [Integrating With Sauce Labs Service Provider](/basics/sso/setting-up-sso/#integrating-with-sauce-labs-service-provider).
```

File: docs/basics/sso/setting-up-sso.md [hunk 4]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
...
 :::
 
 If you enabled encryption of SAML Assertions in your IdP, you must upload the certificate that is provided in Sauce Labs [metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).
 
-The certificate is valid for one year. **120 days** before the expiration date the new certificate will be available in Sauce Labs [metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp). Upload the new certificate in your IdP.
+The certificate is valid for one year. **120 days** before the expiration date the new certificate will be available in Sauce Labs [metadata](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).
+
+During the rotation period the Sauce Labs metadata contain two valid certificates. If you are unsure which certificate to upload, copy the content of the `<ds:X509Certificate>` tag and paste it into [one of the oneline x.509 certificate decoders](https://www.sslshopper.com/certificate-decoder.html). Then, choose the certificate that has a longer validity period.
```
Comment (lysannep) [inline docs/basics/sso/setting-up-sso.md:130]:
```suggestion
During the rotation period, the Sauce Labs metadata contains two valid certificates. If you are unsure which certificate to upload, copy the content of the `<ds:X509Certificate>` tag and paste it into a [certificate decoder](https://www.sslshopper.com/certificate-decoder.html). Then, select the certificate that has a longer validity period.
```

File: docs/basics/sso/setting-up-sso.md [hunk 5]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
...
+
+To convert the certificate from the metadata into the PEM format required by most identity providers, you can utilize the [online tool provided by OneLogin](https://www.samltool.com/format_x509cert.php). Follow these steps:
```
Comment (lysannep) [inline docs/basics/sso/setting-up-sso.md:132]:
```suggestion
To convert the certificate from the metadata into the PEM format required by most identity providers, you can use the [online tool provided by OneLogin](https://www.samltool.com/format_x509cert.php). Follow these steps:
```

File: docs/basics/sso/setting-up-sso.md [hunk 6]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
...
+
+1. Copy the content of the `<ds:X509Certificate>` tag and paste it into the **X.509 cert** text box. Click **FORMAT X.509 CERTIFICATE**.
```
Comment (lysannep) [inline docs/basics/sso/setting-up-sso.md:134]:
```suggestion
1. Copy the content of the `<ds:X509Certificate>` tag, paste it into the **X.509 cert** text box, then click **FORMAT X.509 CERTIFICATE**.
```

File: docs/basics/sso/setting-up-sso.md [hunk 7]
```diff
@@ -108,15 +108,32 @@ The value of the attribute NameID in SAML Response must be **a valid email addre
...
+2. Copy the content of the **X.509 cert with header** textbox and save it in a text file, for example `saucelabs-sso.pem`.
+3. Finally, upload the new certificate in your IdP.
```
Comment (lysannep) [inline docs/basics/sso/setting-up-sso.md:136]:
```suggestion
3. Upload the new certificate in your IdP.
```

**Review by lysannep — APPROVED** [review]
@przemek-sl I added a few suggestions. Thank you!

**Pull request merged**